packages/venia-ui/lib/components/MiniCart2/minicart.css

```
@@ -0,0 +1,85 @@
+.root {
+    /* The root aside element takes up the whole screen. */
+    position: fixed;
+    left: 0;
+    top: 0;
+    height: 100%;
+    width: 100%;
+
+    /* It is hidden by default. */
+    opacity: 0;
+    visibility: hidden;
+
+    /* It animates to being closed. */
+    transition-duration: 192ms;
+    transition-timing-function: var(--venia-anim-out);
+    transition-property: opacity, visibility;
+
+    /* It sits over all background content. */
+    z-index: 3;
+}
+
+.root_open {
+    composes: root;
+
+    opacity: 1;
+    visibility: visible;
+
+    /* It animates to being open. */
+    transition-duration: 224ms;
+    transition-timing-function: var(--venia-anim-in);
+}
+
+.body {
+    margin-top: 1rem;
+    overflow-y: scroll;
+}
+
+.contents {
+    align-content: space-between;
+    background-color: rgb(var(--venia-background-color));
+    border-radius: 1px;
+    box-shadow: 1px 1px 5px rgb(var(--venia-grey-darker));
+    display: grid;
+    /* The header and footer heights may change based on PWA-116 and PWA-601. */
+    grid-template-rows: 2rem 1fr 8rem;
```

It seems more likely that the final row's height should depend on its content. If there's a minimum height, try `minmax()`.

```suggestion
    grid-template-rows: 2rem 1fr minmax(2rem, auto);
```

---

Nice. Resolved in https://github.com/magento/pwa-studio/pull/2494/commits/18d615ed3d7638d1cf9e800b5ea9a586c08601e3.